docs/tutorials/prometheus_locked_down.md

```diff
@@ -1327,9 +1379,12 @@ Verify that the dashboard has been loaded correctly and has the expected configu
 echo "Testing Artemis Broker Metrics dashboard configuration..."
 echo ""
 
+# Get the Grafana admin password from the secret
+GRAFANA_PASSWORD=$(kubectl get secret -n locked-down-broker prometheus-grafana -o jsonpath="{.data.admin-password}" | base64 --decode)
```

Comment:
The previous script already include a similar line to get GRAFANA_PASSWORD, can we export GRAFANA_PASSWORD to avoid a duplicate line?